Source/SwiftLintFramework/Rules/Style/OperatorUsageWhitespaceRuleExamples.swift

```diff
@@ -81,7 +81,12 @@ internal enum OperatorUsageWhitespaceRuleExamples {
         func success(for item: Item) {
             item.successHandler??()
         }
-        """, excludeFromDocumentation: true)
+        """, excludeFromDocumentation: true),
+        Example("""
+        func getAllowedTimeRange(startTime: TimeOfDay) -> TimeOfDayRange {
+            return (startTime)<..<(startTime + 3.hours)
+        }
+        """, configuration: ["additional_allowed_no_space_operators": ["⁝"]])
```

Comment:
Oops, I started by writing an example using the `⁝` operator and then changed my mind but didn't update it properly 😅 

Also it works because it turns out my example of `<..<` is not actually valid swift as you can't use `.` characters in a custom operator. I'll update the example and my PR description 👍